clang/include/clang/Basic/Offloading.h

```diff
@@ -0,0 +1,143 @@
+//===--- Offloading.h - Utilities for offloading ----------------*- C++ -*-===//
```

Comment:
May be OffloadArch.h is a better name. And Description: Definition of offloading architectures and associated utility functions.

Thanks

Reply:
+1 for the name and description suggestion.